zerver/openapi/zulip.yaml

```diff
@@ -11518,6 +11518,24 @@ paths:
                               type: string
                               description: |
                                 The name of the bot.
+                            display_name:
+                              type: string
+                              description: |
+                                The name of the bot that is displayed to the users.
+
+                                **Changes**: New in Zulip 8.0 (feature level 202).
+                            all_event_types:
+                              type: array
+                              items:
+                                type: string
+                              nullable: true
+                              description: |
+                                The event types that are supported by the bot.
```

Comment:
"For incoming webhook integrations that support the Zulip server filtering incoming events, the list of event types supported by the integration. A `null` value will be present if this is not an incoming webhook integration."

Reply:
Yeah, this describes the whole thing precisely.